lib/administrate/field/base.rb

```diff
@@ -27,6 +27,10 @@ def self.permitted_attribute(attr)
         attr
       end
 
+      def label
+        options[:label] || attribute.to_s.humanize.capitalize
```

Comment:
What if we allowed the overriding but didn't attempt to figure out the label name from the attribute if the option was missing and instead just returned `nil`? Then we can continue to let the form builder handle the unspecified case.